### ISSUE TITLE
[web] Back button should erase what's written

### ISSUE THREAD

**Issue by danielaRiesgo** [Festify/app#274]
Steps to reproduce:
1. Write a song/artist in search box
2. Click back button
3. You get back to playlist but what you searched for stays written

I think when pressing back it should erase what you searched for.

**Comment by NeoLegends** [comment]
Yeah that‘s a bug. It used to do that before. Thanks for the report!
